src/pages/index.js

```diff
@@ -137,18 +215,67 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitBtnTwo = evt.submitter;
+  setButtonText(submitBtnTwo, true, "Saving...", "Save");
+  //submitBtnTwo.textContent = "Saving...";
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      profileAvatar.src = data.avatar;
+      evt.target.reset();
+      disableButton(submitBtn, config.inactiveButtonClass);
+      closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtnTwo, false, "Saving...", "Save");
+    });
 }
 ////////////////////////////////////
 function handleAddCardSubmit(evt) {
   evt.preventDefault();
+  const submitButton = evt.submitter;
+
   const inputValues = { name: cardNameInput.value, link: cardLinkInput.value };
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  evt.target.reset();
-  closeModal(cardModal);
+  setButtonText(submitButton, true, "Save", "Saving...");
+  api
+    .addCard(inputValues)
+    .then((data) => {
+      const cardElement = getCardElement(data);
+      cardsList.prepend(cardElement);
+      evt.target.reset();
```

Comment:
Please, add ` disableButton(submitButton, config.inactiveButtonClass);` in line 251 to disable the card save button after submission